### PR TITLE
added hw_3

### PR DESCRIPTION
**Pull request by marqshee**

### CODE REVIEW

**Comment by Borgaard** [comment]
@Divii Very nice hash exercises. I'll leave this open for you to push `car_lot.rb`.

**Comment by marqshee** [comment]
@Borgaard car_lot.rb is in hw_2 directory.

**Comment by marqshee** [comment]
This PR now holds get_it.rb

File: sheenamarquez-sheena/hw_4/get_it.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    puts "#{network}: #{slug}"
+    index += 1
+  end
+end
+
+def record_stories(content)
+  CSV.open('records.cvs', 'a+') do |cvs|
+    cvs << content
+  end
+end
```
Comment (Borgaard) [inline sheenamarquez-sheena/hw_4/get_it.rb:85]: Oooo fancy CSVs! Today we'll be diving a little further in depth with APIs. I'm thrilled to see you've taken a lot of agency with this project in using a different API and HTTP gem, and encourage you to continue coding with curiosity in future projects. 🐙

**Pull request merged**